panther_analysis_tool/backend/public_api_client.py

```diff
@@ -106,7 +110,33 @@ def check(self) -> BackendCheckResponse:
         )
 
     def bulk_upload(self, params: BulkUploadParams) -> BackendResponse:
-        pass
+        query = self._requests.bulk_upload_mutation()
+        upload_params = {"input": {"data": params.encoded_bytes()}}
+        default_stats = dict(total=0, new=0, modified=0)
+        res = res = self._execute(query, variable_values=upload_params)
+
+        if res.errors:
+            for err in res.errors:
+                logging.error(err.message)
+
+            return BackendCheckResponse(
```

Comment:
`BackendCheckResponse` was meant to be a special value that only came from `def check()`. We could probably eliminate it and use the generic `BackendResponse[CheckResponse]` format instead 

Might want to extend `BackendResponse` to have an error field?

Reply:
woah good catch